Wire-iOS/Sources/UserInterface/Components/MessagePreviewView.swift

```diff
@@ -224,11 +222,22 @@ final class MessagePreviewView: UIView {
             ])
     }
     
+    private func editIcon() -> NSAttributedString {
+        if message.updatedAt != nil {
+            return "  " + NSAttributedString(attachment: NSTextAttachment.textAttachment(for: .pencil, with: .textForeground, iconSize: 8)!)
+        }
+        else {
+            return NSAttributedString()
+        }
+    }
+    
     private func updateForMessage() {
-        senderLabel.text = message.sender?.displayName(in: message.conversation!)
+        
         let attributes: [NSAttributedString.Key: Any] = [.font: UIFont.smallSemiboldFont,
                                                          .foregroundColor: UIColor.textForeground]
         
+        senderLabel.attributedText = ((message.sender?.displayName(in: message.conversation!) ?? "") && attributes) + self.editIcon()
```

Comment:
We added the `senderName` utility property with the cell refactoring and it could be easier to use it here.

It can compute the sender name in the conversation, and also takes into account the self user (it returns "You") and null sender (it returns "Someone").

```suggestion
        senderLabel.attributedText = (message.senderName && attributes) + self.editIcon()
```